instagram/src/App.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import './App.css';
+
+// Import Components
+
+import dummyData from './dummy-data';
+import SearchBar from './components/SearchBar/SearchBar'
+import PostContainer from './components/PostContainer/PostContainer';
+
+// Import CSS
+
+import './components/SearchBar/searchBar.css';
+import './components/CommentSection/commentSection.css';
+import './components/PostContainer/postContainer.css';
+import './components/CommentSection/commentSection.css'
+
+// App Constructor
+
+class App extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      dummyData: dummyData
+    }
```

Comment:
Nicely done. I love that you're keeping your data in App level. This will help you when you're working with search functionality later.